tests/functional/c/comparison_of_constants.py

```diff
@@ -22,14 +18,6 @@
 
 CONST = 24
 
-if CONST is 0:  # [literal-comparison]
```

Comment:
Why did you remove these?

Reply:
It's duplicated from the literal comparison functional tests.